web/src/nav/StatusMessagesNavItem.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { setLinkComponent } from '../../../shared/src/components/Link'
+import * as GQL from '../../../shared/src/graphql/schema'
+import { StatusMessagesNavItem } from './StatusMessagesNavItem'
+
+describe('StatusMessagesNavItem', () => {
+    setLinkComponent((props: any) => <a {...props} />)
```

Comment:
I don't get a compile error when removing the `any`, so it seems unnecessary 
```suggestion
    setLinkComponent(props => <a {...props} />)
```